auth/src/main/java/org/openedx/auth/presentation/signin/compose/SignInView.kt

```diff
@@ -276,7 +291,9 @@ private fun SocialLoginView(
 ) {
     if (state.isGoogleAuthEnabled) {
         OpenEdXOutlinedButton(
-            modifier = buttonWidth.padding(top = 24.dp),
+            modifier = buttonWidth
+                .testTag("btn_google_auth")
```

Comment:
Would providing the ID to the Text Field inside the Row container be beneficial for Bilal? Same goes for other Social Buttons as well.

Reply:
yes, requested by the Automation team. assign the id to the Text.